two_factor/forms.py

```diff
@@ -169,6 +170,14 @@ def __init__(self, user, initial_device, **kwargs):
                 isinstance(initial_device, (RemoteYubikeyDevice, YubikeyDevice)):
             self.fields['otp_token'] = forms.CharField(label=_('YubiKey'), widget=forms.PasswordInput())
 
+        # Add a field to remeber this browser.
+        if getattr(settings, 'TWO_FACTOR_REMEMBER_COOKIE_AGE', None):
+            self.fields['remember'] = forms.BooleanField(
+                required=False,
+                initial=True,
+                label=_("Don't ask again on this device")
```

Comment:
I think the user needs to be informed how long their second factor will be remembered for. Especially important on the example site where it's only going to be remembered for 60 seconds :laughing:

Reply:
I agree!